pro/src/components/pages/Offerers/Offerer/VenueV1/fields/PricingPointV2/PricingPointV2.module.scss

```
@@ -0,0 +1,39 @@
+@use 'styles/mixins/_rem.scss' as rem;
+
+.title {
+  font-size: rem.torem(16px);
+  font-weight: 600;
+}
+
+.reimbursement-subtitle {
+  margin: rem.torem(24px) 0;
+  font-size: rem.torem(15px);
+
+  .text-hightlight {
+    font-weight: 600;
+  }
+}
+
+.text-dialog {
+  margin-bottom: rem.torem(30px);
+}
+
+.dropDown-container {
```

```suggestion
.dropdown-container {
```
pour ne pas avoir de mélange de typo :)